tools/buildsteps/freebsd/make-depends

```diff
@@ -2,4 +2,7 @@ WORKSPACE=${WORKSPACE:-$( cd $(dirname $0)/../../.. ; pwd -P )}
 XBMC_PLATFORM_DIR=freebsd
 . $WORKSPACE/tools/buildsteps/defaultenv
 
-#nothing for freebsd atm
+if [ "$(pathChanged $WORKSPACE/tools/depends)" == "1" ]
+then
+  cd $WORKSPACE/tools/depends;make -C target/libfstrcmp -j $BUILDTHREADS && tagSuccessFulBuild $WORKSPACE/tools/depends
```

Comment:
that doesn't work

Reply:
i suspected as much, i didn't have anything to monkey.